src/Symfony/Component/HttpFoundation/Session/Storage/Handler/SessionHandlerWrapper.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
+
+/**
+ * Adds basic `SessionUpdateTimestampHandlerInterface` behaviors to another `SessionHandlerInterface`.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class SessionHandlerWrapper extends AbstractSessionHandler
```

Comment:
can't we come up with an explicit name like `SessionFixationPreventionAdapter`

Reply:
Pushed as `StrictSessionHandler`, docblock added on `AbstractSessionHandler`.